gen_cert.py

```diff
@@ -2065,6 +2065,12 @@ def _generate_v4_certificate(
                             continue
                         draw_template_element(self, element, attributes, page)
 
+        # Render any badges
```

Comment:
Should this happen before we render the course information or after we've rendered everything? Why?